packages/calendar/src/Calendar.ts

```diff
@@ -0,0 +1,449 @@
+                tabindex=${ifDefined(!this.disabled ? '0' : undefined)}
+                aria-readonly="true"
+                aria-disabled=${this.disabled}
+            >
+                <table role="presentation" class="spectrum-Calendar-table">
+                    ${this.renderCalendarTableHead()}
+                    ${this.renderCalendarTableBody()}
+                </table>
+            </div>
+        `;
+    }
+
+    public renderCalendarTableHead(): TemplateResult {
+        return html`
+            <thead role="presentation">
+                <tr role="row">
+                    ${this._getWeekdays().map((weekday) =>
+                        this.renderWeekdayColumn(weekday)
+                    )}
+                </tr>
+            </thead>
+        `;
+    }
+
+    public renderWeekdayColumn(weekday: CalendarWeekday): TemplateResult {
+        return html`
+            <th
+                role="columnheader"
+                scope="col"
+                class="spectrum-Calendar-tableCell"
+            >
+                <abbr class="spectrum-Calendar-dayOfWeek" title=${weekday.long}>
+                    ${weekday.narrow}
+                </abbr>
+            </th>
+        `;
+    }
+
+    public renderCalendarTableBody(): TemplateResult {
+        return html`
+            <tbody role="presentation">
+                ${[...new Array(this._getWeeksInCurrentMonth()).keys()].map(
+                    (weekIndex) => this.renderCalendarTableRow(weekIndex)
+                )}
+            </tbody>
+        `;
+    }
+
+    public renderCalendarTableRow(weekIndex: number): TemplateResult {
+        return html`
+            <tr role="row">
+                ${this._getDatesInWeek(weekIndex).map((calendarDate) =>
+                    this.renderCalendarTableCell(calendarDate)
+                )}
+            </tr>
+        `;
+    }
+
+    public renderCalendarTableCell(calendarDate: CalendarDate): TemplateResult {
+        const isOutsideMonth = calendarDate.month !== this._currentDate.month;
+
+        const isSelected = Boolean(
+            this.selectedDate &&
+                isSameDay(this._toCalendarDate(this.selectedDate), calendarDate)
+        );
+
+        const isToday = Boolean(isSameDay(calendarDate, this.today));
+
+        const isDisabled = Boolean(
+            this.disabled ||
+                (this._minDate && calendarDate.compare(this._minDate) < 0) ||
+                (this._maxDate && calendarDate.compare(this._maxDate) > 0)
+        );
+
+        const dayClasses = {
+            'is-outsideMonth': isOutsideMonth,
+            'is-selected': isSelected,
+            'is-today': isToday,
+            'is-disabled': isDisabled,
+        };
+
+        const currentDayTitle = this._formatDate(calendarDate, {
+            weekday: 'long',
+            year: 'numeric',
+            month: 'long',
+            day: 'numeric',
+        });
+
+        // TODO: Translate "Today" and "selected"
```

Comment:
Something you can try here. I created a pen for you for the translation logic
https://codesandbox.io/s/infallible-khayyam-cnymcs?file=/src/index.js
library used i18next

Reply:
Thanks for the example, Raj. At this time I will not include the translations, we will do this when the component is finalised and then all TODOs will be resolved so we can then merge the feature branch into main